content/docs/guides/pipelines/pipelines-metrics.md

```diff
@@ -0,0 +1,35 @@
++++
+title = "Pipeline metrics"
+description = "Export and visualize pipeline metrics."
+weight = 3
+toc = true
+
+[menu.docs]
+  parent = "pipelines"
+  weight = 6
++++
+This page is for component author to export metrics from the component. For details about how to build your own component, please see [Editing Build Your Own Component](/docs/guides/pipelines/build-component).
+ 
+## Overview of Metrics
```

Comment:
For this heading and all those below: 

For code readability, please add an empty line after the heading.

Let's use capital letters only for brand names. So, headings should start with a capital letter but use lower case for the rest of the heading.

Reply:
done